app/views/posts/stats.haml

```diff
@@ -40,7 +40,7 @@
     %tr
       %th.sub.width-150 Authors
       %td{class: cycle('odd', 'even')}
-        - post_authors = @post.post_authors.joins(:user).order(Arel.sql('LOWER(users.username) asc'))
+        - post_authors = @post.post_authors.joins(:user).includes(:user).order(Arel.sql('LOWER(users.username) asc'))
```

Comment:
Post stats page uses an n+1 to figure out how to render users - joins doesn't do an includes for purposes of the associations, so we still do an n+1 fetch afterwards